plugins/inventory/foreman.py

```diff
@@ -274,6 +289,17 @@ def _populate(self):
                             except ValueError as e:
                                 self.display.warning("Could not create groups for host collections for %s, skipping: %s" % (host_name, to_text(e)))
 
+                # put ansible_ssh_host as hostvar
+                if self.get_option('want_ansible_ssh_host'):
+                    for key in ('ip', 'ipv4', 'ipv6'):
+                        if host.get(key):
+                            try:
+                                self.inventory.set_variable(host_name, 'ansible_ssh_host', host[key])
```

Comment:
isn't ansible_host preferred? I'm not particular, I just want to raise the question early.